integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java

```diff
@@ -126,7 +121,7 @@ public void shouldSubscribeToProcessUpdatesNotification() throws Exception {
         //then
         logger.info("Received: " + notificationCollector.getMessages().get(0));
 
-        assertThat(notificationCollector.getMessages().get(0)).startsWith("{\"oldStatus\":\"NEW\",\"build\":{\"id\":\"1\",\"status\":\"SUCCESS\",");
+        assertTrue(notificationCollector.getMessages().get(0).startsWith("{\"oldStatus\":\"NEW\",\"build\":{\"id\":\"1\",\"status\":\"SUCCESS\","));
```

Comment:
why did you changed it from assetj to just junit?

Reply:
There was a classloading issue with assertj, which is why I just replaced it with JUnit

Reply:
only in this one file?

Reply:
Yup. It's also IIRC the only file that uses it.